docs/faq.md

```diff
@@ -103,26 +103,26 @@ See [Extended Usage](../jib-gradle-plugin#extended-usage) for the `container.for
 
 ### Why is my image created 48+ years ago?
 
-For reproducibility purposes, Jib sets the creation time of the container images to 1 second past the Unix epoch (00:00:01, January 1st, 1970 in UTC). If you would like to forgo reproducibility and use the real creation time, set the `useCurrentTimestamp` parameter to `true` in your build configuration.
+For reproducibility purposes, Jib sets the creation time of the container images to the Unix epoch (00:00:00, January 1st, 1970 in UTC). If you would like to forgo reproducibility and use the real creation time, set the `jib.container.creationTime` / `<container><creationTime>` parameter to `USE_CURRENT_TIMESTAMP` in your build configuration.
```

Comment:
I think we should promote setting a fixed timestamp instead of `USE_CURRENT_TIMESTAMP`, although mentioning `USE_CURRENT_TIMESTAMP` is OK and meaningful with the reproducibility warning.